### DISCUSSION TITLE
Is OpenCart sticking to 3.0.3.9 with OcMod? or 4.0.X.X with Events?

### DISCUSSION THREAD

**Discussion by ansleyD** [opencart/opencart#13780] [General]
Hi,

I am creating a new store with many extensions and modifications to make changes to. And I want to know which version of OC the development team adopted 3.0.X.X or 4.0.X.X? So I don't stuck up in a version that is not developing.

Regards

**Comment by felipoantonoff** [comment]
The one that will be supported for a longer period of time should be 4, but I would wait for the OCMOD to arrive on it or you can start with the current one with vQmod and then when the native OCMOD returns, switch to it.

The current Opencart 4 does not have OCMOD and Events will never replace OCMOD, as the scope and uses are different, basically OCMOD can do everything that Events does and much more, but Events is basically a trigger. Each one should be used to its best advantage, if you want to change many files such as native files, themes, etc., OCMOD is the best tool for this and best of all, except for changes in cache, they are also better in terms of performance.

3.0.3.9 is the most tested and mature version of Opencart, it will also have more modules. But the tendency is for Opencart 4 to have more modules in the future as it will become the most used over time.
